build-scripts/webpack.js

```diff
@@ -29,7 +29,7 @@ const createWebpackConfig = ({
     module: {
       rules: [
         {
-          test: /\.js$|\.ts$/,
+          test: /\..?js$\.ts$/,
```

Comment:
Are you sure about this? We used to apply Babel to anything ending in js or ts, you dropped the or-symbol?

Reply:
Oh shit, the or should not have been dropped. I should just match `.mjs` too